www/include/options/accessLists/resourcesACL/DB-Func.php

```diff
@@ -452,15 +486,20 @@ function updatePollers($acl_id = null)
         return;
     }
 
-    $pearDB->query("DELETE FROM acl_resources_poller_relations WHERE acl_res_id = '" . $acl_id . "'");
+    $statement = $pearDB->prepare("DELETE FROM acl_resources_poller_relations WHERE acl_res_id = :acl_res_id");
+    $statement->bindValue(':acl_res_id', (int)$acl_id, \PDO::PARAM_INT);
+    $statement->execute();
     $ret = array();
     $ret = $form->getSubmitValue("acl_pollers");
     if (isset($ret)) {
         foreach ($ret as $key => $value) {
             if (isset($value)) {
-                $query = "INSERT INTO acl_resources_poller_relations (acl_res_id, poller_id) VALUES ('" .
-                    $acl_id . "', '" . $value . "')";
-                $res = $pearDB->query($query);
+                $query = "INSERT INTO acl_resources_poller_relations (acl_res_id, poller_id) VALUES (:acl_res_id, 
+                                                                           :poller_id)";
+                $statement = $pearDB->prepare($query);
```

Comment:
do not prepare query in foreach

Reply:
@kduret you're right i missed that my bad :)